package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gitlab",
   "description": "Full NodeJS implementation of the GitLab API. Supports Promises, Async/Await.",
-  "version": "14.0.0",
+  "version": "14.0.1",
```

Comment:
seems odd that this is touching version 🤔

Reply:
Just because i was pushing to a beta tag on npm